src/components/EmptyState.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Box, Center } from "@chakra-ui/react";
+
+type EmptyStateProps = {
+    styleProps?: Record<string, unknown>;
+};
+
+export const EmptyState: React.FC<EmptyStateProps> = ({
+    children,
+}): JSX.Element => {
+    return (
+        <Box bg={"#F8F8F8"} w="100%" py={"40px"} px={"64px"} color={"#737373"}>
```

Comment:
I think we might want to add these colours into `src/styles/colours.ts` and use them from our colour palette. For example, look at how it is being used in the `volunteer-signup` page.
We might endup with the following:
```suggestion
        <Box bg={colourTheme.colors.Gray} w="100%" py={"40px"} px={"64px"} color={colourTheme.colors.LightGray}>
```